hypervisor/src/kvm/mod.rs

```diff
@@ -407,7 +407,10 @@ impl hypervisor::Hypervisor for KvmHypervisor {
 
         #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
         {
-            Ok(Arc::new(KvmVm { fd: vm_fd }))
+            Ok(Arc::new(KvmVm {
+                fd: vm_fd,
+                state: VmState {},
+            }))
```

Comment:
I'd suggest this can be done as part of the global enabling of `VmState`, you don't need a dedicated commit for `aarch64` as the change is very trivial.